edx-solutions/xblock-drag-and-drop-v2
# Added publishing progress event no matter if grade is updated or not

**Description:** This PR makes DnDv2 always publish progress event when grading occurs, no matter if grade have improved or not.

**JIRA:** This is an attempt to fix [MCKIN-6011](https://edx-wiki.atlassian.net/browse/MCKIN-6011)

**Testing instructions:**

All the normal operations should happen as usual. This code potentially fixes one particular use case:

1. Create new DnDv2
2. Fake StudentModule data for that XBlock to contain raw_score: 1.0
3. Open block in Apros, submit the solution
4. Make sure Apros progress indicators (on module page and on progress page) are updated after page refresh.

**Reviewers:**
